roll/testing.py

```diff
@@ -62,12 +65,16 @@ class Transport:
 
     def __init__(self):
         self.data = b''
+        self._closing = False
+
+    def is_closing(self):
```

Comment:
Are we using this in some way?

Reply:
We don't, but i think a mock Transport should have the basic methods a Transport has, this was a start.